server/data/activeLduClient.js

```diff
@@ -1,19 +1,19 @@
 const db = require('./dataAccess/db')
 
 module.exports = {
-  async addLdu(lduCode) {
+  async addLdu(lduCode, probationAreaCode) {
```

Comment:
meh.. would generally order arguments in terms of size - as probation area is the more general, further arguments providing more context.